_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -166,3 +166,17 @@ than the max age.
 For examples of how to define scanner rules for your service account keys, see the
 [`service_account_key_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/service_account_key_rules.yaml)
 file.
+
+
+## External Project Access Scanner
+
+The External Project Access Scanner mitigates data exfiltration by identifying users who have access to projects outside of your organization. 
+
+## What makes this scanner different?
+Each user in the inventory must be queried for their project access. The number of users in an organization will impact the execution time of this scanner.  It may therefore be undesirable to execute this scanner as frequently as other scanners.  By default, this scanner is not enabled in the Forseti server configuration.
+
+In the first release of this scanner, it is invoked manually on either the client or server vm in GCP as follows:
+`forseti scanner run --scanner external_project_access_scanner`
+
+Before running this scanner, please make sure your service account has the following API scope:
+`https://www.googleapis.com/auth/cloudplatformprojects.readonly`
```

Comment:
Instead of referencing this explicitly, can you make this reference to this link?  Please use an internal ref link.
```
https://forsetisecurity.org/docs/latest/configure/inventory/gsuite.html#enable-the-service-account-in-your-g-suite-admin-control-panel
```
internal ref link
```
[automated installer]({% link _docs/latest/setup/install.md %})

```

Reply:
modified